### DISCUSSION TITLE
Caprover install

### DISCUSSION THREAD

**Discussion by mathiasfoster** [pocketbase/pocketbase#65] [Q&A]
Good morning,

Is there any way for PocketBase to be installed on Caprover?
I found this page, but couldnt make sense of it!
https://caprover.com/docs/sample-apps.html

**Comment by ganigeorgiev** [comment]
I haven't use CapRover, but at first glance it seems to rely on Docker, so I guess you may need to create a Dockerfile.

Currently there is no official Docker image (see #64), but you can always create your own as downloading the linux version from the [releases page](https://github.com/pocketbase/pocketbase/releases), extracting and executing the binary inside it is all you need to get PocketBase running.
